libexec/configure/actions/dropcache

```diff
@@ -0,0 +1,58 @@
+#!/bin/bash
+#==============================================================================
+# Copyright (C) 2017 Stephen F. Norledge and Alces Software Ltd.
+#
+# This file/package is part of Alces Clusterware.
+#
+# Alces Clusterware is free software: you can redistribute it and/or
+# modify it under the terms of the GNU Affero General Public License
+# as published by the Free Software Foundation, either version 3 of
+# the License, or (at your option) any later version.
+#
+# Alces Clusterware is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+# Affero General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with this package.  If not, see <http://www.gnu.org/licenses/>.
+#
+# For more information on the Alces Clusterware, please visit:
+# https://github.com/alces-software/clusterware
+#==============================================================================
+require process
+require action
+
+_dropcache_clear_cache() {
+    local cache_int="$1"
+    shift
+
+    echo "$cache_int" > /proc/sys/vm/drop_caches
+}
+
+main() {
+    local input="$1"
+    shift
+
+    case "$input" in
+        "")
+            $cw_ROOT/bin/alces configure help dropcache
+            ;;
+        1|p|pa|pag|page|pagec|pageca|pagecac|pagecac|pagecach|pagecache)
```

Comment:
We should drop the `1`, `2` and `3` from here too for consistency.

Also, I think it would be better UX to emit a message once the operation has been performed.  Perhaps:

```
action_warn "clean caches from pagecache dropped"
```

...Along with: "reclaimable slab objects dropped" and "clean caches from pagecache and reclaimable slab objects dropped".

What do you reckon?